pkg/apis/core/types.go

```diff
@@ -2891,6 +2907,16 @@ type PodSecurityContext struct {
 	// sysctls (by the container runtime) might fail to launch.
 	// +optional
 	Sysctls []Sysctl
+	// Defines behavior of changing SELinux labels of the volume before being exposed inside Pod
+	// (Alpha feature).
+	// This field is ignored for Pod's volumes and/or for Linux installations that do not support
+	// SELinux.
+	// Valid values are "OnVolumeMount" and "Always". If not specified, "Always" is used.
+	// "Always" policy recursively changes SELinux labels on all files on all volumes used by the Pod.
+	// "OnVolumeMount" tries to mount volumes used by the Pod with the right context and skip recursive ownership
+	// change. Kubernetes may fall back to policy "Always" if a storage backed does not support this policy.
+	// + optional
+	SELinuxRelabelPolicy *PodSELinuxRelabelPolicy
```

Comment:
I can't remember if this came up on the KEP, but should this be per-volume rather than per-pod?

Reply:
See below, I'm following FSGroupChangePolicy.